package.json

```diff
@@ -10,17 +10,25 @@
   "author": "",
   "license": "ISC",
   "devDependencies": {
+    "@babel/cli": "^7.21.5",
+    "@babel/core": "^7.21.8",
     "babel-eslint": "^10.1.0",
+    "babel-loader": "^9.1.2",
     "eslint": "^7.32.0",
     "eslint-config-airbnb-base": "^14.2.1",
     "eslint-plugin-import": "^2.27.5",
     "hint": "^7.1.8",
     "html-webpack-plugin": "^5.5.1",
+    "lodash": "^4.17.21",
+    "style-loader": "^3.3.2",
     "stylelint": "^13.13.1",
     "stylelint-config-standard": "^21.0.0",
     "stylelint-csstree-validator": "^1.9.0",
     "stylelint-scss": "^3.21.0",
     "webpack": "^5.82.1",
     "webpack-cli": "^5.1.1"
```

Comment:
- [ ] kindly consider installing `webpack-dev-server` as it provides a convenient development server for running your `webpack-built` project. By using `webpack-dev-server`, you can benefit from features such as live reloading, and a simple setup for development purposes.